cc/cc_toolchain_config_lib.bzl

```diff
@@ -446,17 +454,23 @@ def tool_path(name, path):
     _check_is_nonempty_string(path, "path", "tool_path")
     return ToolPathInfo(name = name, path = path, type_name = "tool_path")
 
-ToolInfo = provider(
-    "Describes a tool associated with a crosstool action config.",
-    fields = ["path", "with_features", "execution_requirements", "type_name"],
-)
+ToolInfo = provider(fields = [
```

Comment:
If you add a docstring right before `fields` I think buildifier should stop complaining.

Reply:
Thanks, that fixed the buildifier failure :)